var/spack/repos/builtin/packages/netcdf-cxx4/package.py

```diff
@@ -19,61 +21,122 @@ class NetcdfCxx4(AutotoolsPackage):
     version('4.3.1', sha256='6a1189a181eed043b5859e15d5c080c30d0e107406fbb212c8fb9814e90f3445')
     version('4.3.0', sha256='e34fbc6aba243ec82c23e9ee99db2430555ada849c54c1f3ab081b0ddd0f5f30')
 
-    # Usually the configure automatically inserts the pic flags, but we can
-    # force its usage with this variant.
-    variant('static', default=True, description='Enable building static libraries')
```

Comment:
(question) With this change, does `+shared` build both the shared and static libraries, or just the shared libraries?

Reply:
@scheibelp the static libraries are always built and the shared only when `+shared`